tests/python/rest_api/test_jobs.py

```diff
@@ -567,16 +569,18 @@ def test_get_gt_job_in_org_task(
             self._test_get_job_403(user["username"], job["id"])
 
 
-@pytest.mark.usefixtures(
-    # if the db is restored per test, there are conflicts with the server data cache
-    # if we don't clean the db, the gt jobs created will be reused, and their
-    # ids won't conflict
-    "restore_db_per_class"
-)
```

Comment:
Why these lines were removed?

Reply:
This was restoring the db state. But in the tests here we are ensuring that the db state is restored by deleting the new objects created (which is only gt job here). So thought to remove, but if you say that there might be risk of db state getting changed I shall again add it.